docker/pytorch/Dockerfile

```diff
@@ -10,6 +10,18 @@ RUN /opt/conda/bin/conda install tensorflow-gpu==2.2.0 \
  cudatoolkit=10.1 -c pytorch && \
     /opt/conda/bin/conda clean --all
 
+ENV NUMBA_CACHE_DIR /tmp
+ENV LD_LIBRARY_PATH="${LD_LIBRARY_PATH}:/usr/local/cuda/lib64"
+
+RUN /opt/conda/bin/conda install -c conda-forge librosa 
+RUN /opt/conda/bin/pip install hydra-core==1.0.0rc1 \
+  python-levenshtein \
+  torchaudio \
+  numba==0.43.0 \ 
+  --ignore-installed llvmlite==0.32.1 \ 
+  soundfile \ 
+  sox \ 
+  warpctc_pytorch
```

Comment:
Are all of these additional dependencies required? Which are due to the attack and which are due to the model?

Reply:
These are all dependencies for the model. These are specifically for the deepspeech.pytorch package (https://github.com/SeanNaren/deepspeech.pytorch) which is what ART uses for in their implementation of DeepSpeech.